etc/osg/image-config.d/20-osg-ce-setup.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+. ce-common-startup
```

Comment:
Note that, i think, this will source the `ce-common-startup` from whatever directory you are running this 20-osg-ce-setup.sh script; i don't think it's automatic to look relative to the script that's running.